websvc.metro.lib/src/org/netbeans/modules/websvc/metro/lib/metro.xml

```diff
@@ -36,13 +36,22 @@
     </volume>
     <volume>
         <type>javadoc</type>
-        <resource>jar:nbinst://org.netbeans.modules.websvc.metro.lib/docs/metro-1.4-doc.zip!/</resource>
+        <resource>jar:nbinst://org.netbeans.modules.websvc.metro.lib/docs/webservices-api-javadoc.jar!/</resource>
+        <resource>jar:nbinst://org.netbeans.modules.websvc.metro.lib/docs/webservices-extra-javadoc.jar!/</resource>
+        <resource>jar:nbinst://org.netbeans.modules.websvc.metro.lib/docs/webservices-extra-api-javadoc.jar!/</resource-->
```

Comment:
The closing tag is wrong - \</resource--> should be \</resource>

Reply:
Corrected, Thanks